### PR TITLE
Pandaren portraits

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Added Pandaren portraits for both genders

## Developer changelog:
- Pandarens only have one hair/fur color.

# How to test:
- Check portraits in ruler designer

### CODE REVIEW

File: interface/portraits/portraits_pandaren_sprites.gfx [hunk 1]
```diff
@@ -21,17 +21,107 @@ spriteTypes = {
...
+	
+	spriteType = {
+		name = "GFX_pandaren_male_beard"
+		texturefile = "gfx\\characters\\pandaren_male\\pandaren_male_beard_1.dds"
+		noOfFrames = 21
+		norefcount = yes
+	}
+	
+	spriteType = {
+		name = "GFX_pandaren_male_ear"
```
Comment (zumbak04) [inline interface/portraits/portraits_pandaren_sprites.gfx:64]: Shouldn't it be eyes layer?

**Comment by zumbak04** [comment]
Many females don't have hairs.
<details>
<summary>Click to expand</summary>

image

</details>

File: interface/portraits/portraits_pandaren.gfx [hunk 2]
```diff
@@ -11,26 +11,35 @@ spriteTypes = {
...
+			"GFX_pandaren_male_headgear:p5"
 			
 			"GFX_worgen_druid_form:p19"
 			"GFX_character_imprisoned:p6"
 			"GFX_being_overlay:p20"
 			"GFX_player_overlay:p11"
 		}
 
-		hair_color_index = 8
+		hair_color_index = 1
```
Comment (zumbak04) [inline interface/portraits/portraits_pandaren.gfx:32]: Index changed.

File: interface/portraits/portraits_pandaren.gfx [hunk 3]
```diff
@@ -11,26 +11,35 @@ spriteTypes = {
...
+			{ 0 0 0 } {  75 75 75 } { 255 255 255 } # Black
 		}
 
 		eye_color_index = 9
 		eye_color = {
 			{ 0 0 0 } {  125 125 125 } { 255 255 255 }	# Default gray
 		}
 
-		headgear_that_hides_hair = { }
+		headgear_that_hides_hair = { 11 }
```
Comment (zumbak04) [inline interface/portraits/portraits_pandaren.gfx:42]: There are 3 headgears that should hide hairs.
<details>
<summary>Click to expand</summary>

image

</details>

File: interface/portraits/portraits_pandaren_sprites.gfx (hunk 1 above)
Comment (sejtalk) [reply interface/portraits/portraits_pandaren_sprites.gfx]: does it matter?

**Comment by sejtalk** [comment]
> Many females don't have hairs.
> 
> Click to expand

fixed

**Review by Myrran — COMMENTED** [review]
- Male pandas look fine
- Female pandas look fine
- Emperor clothes for pandaren males are fine

Notes:

- Eyes change color when changing faces (neck)
- I am not sure about the hat + bandana headgear (#10). I think pandas lose their individuality and look similar to one another when wearing these

ck2_1

File: interface/portraits/portraits_pandaren_sprites.gfx (hunk 1 above)
Comment (zumbak04) [reply interface/portraits/portraits_pandaren_sprites.gfx]: > does it matter?

Yes.

**Comment by zumbak04** [comment]
> * I am not sure about the hat + bandana headgear (#10). I think pandas lose their individuality and look similar to one another when wearing these

It's a martial outfit. It usually covers the face so there's a game rule to disable it on rulers.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Comment by sejtalk** [comment]
> * Male pandas look fine
> * Female pandas look fine
> * Emperor clothes for pandaren males are fine
> 
> Notes:
> 
> * Eyes change color when changing faces (neck)
> * I am not sure about the hat + bandana headgear (#10). I think pandas lose their individuality and look similar to one another when wearing these
> 
> ck2_1

eye color changing is on purpose and i think bandana is fine tbh

**Review by Myrran — APPROVED** [review]
okay then, approved.

**Pull request merged**